### DISCUSSION TITLE
Support d’assistance Sms : - Contrôler les effets spaghetti avec ggplot

### DISCUSSION THREAD

**Discussion by giscus** [mthevenin/assistoolsms#12] [General]
# Support d’assistance Sms : - Contrôler les effets spaghetti avec ggplot

https://mthevenin.github.io/assistoolsms/R/assist/posts/spaghetti/spaghetti.html

**Comment by mthevenin** [comment]
* J'ai repris un peu la forme du doc. 
* Pour la maj de la fonction + explication de la fonction, c'est jouable en mars?

**Reply by mthevenin** [reply]
Nickel !!!!
